### DISCUSSION TITLE
Change/Hide Shortcut (app icon) in launcher

### DISCUSSION THREAD

**Discussion by VixenSugo** [Mahmud0808/Iconify#465] [Ideas]
can you add this feature?
icons (shortcuts) saved in this path: /data/data/... (root directory)
thanks for this project.

**Comment by tomaszk8266** [comment]
It's a feature request, they should be submitted in issues tab

**Comment by VixenSugo** [comment]
> It's a feature request, they should be submitted in issues tab

you're right
Sorry , it was careless of me.
